src/components/Settings.vue

```diff
@@ -44,18 +44,30 @@
         >
           {{ t('memories', 'Show past photos on top of timeline') }}
         </NcCheckboxRadioSwitch>
+      </NcAppSettingsSection>
+
+      <NcAppSettingsSection id="preview-settings" :title="t('memories', 'Previews')">
+        <NcCheckboxRadioSwitch
+          :checked.sync="config.override_global_full_res"
+          @update:checked="updateOverrideGlobalFullRes"
+          type="switch"
+        >
+          {{ t('memories', 'Override the global settings for loading full size images') }}
```

Comment:
Don't think we need this; seems too complex for the average user. Just a default provided by the admin is good enough.

Anyway, this looks good overall, will make some adjustments before merging.

Reply:
I added this because I wasn't sure how otherwise a user could turn off the default, but I assume you are thinking that if an admin has turned it on then there's not a good reason that an individual user would need to turn it off?

Also would you like me to remove this or will you be doing it?